Shear/AppDelegate.swift

```diff
@@ -14,12 +14,13 @@ class AppDelegate: NSObject, NSApplicationDelegate {
     @IBOutlet weak var window: NSWindow!
     
     var app: App!
-
+    
     func applicationDidFinishLaunching(_ aNotification: Notification) {
-        app = App()
+        let history = History(storage: InMemoryHistoryStorage())
+        app = App(history: history)
         app.start()
     }
-
+    
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/kmpnz/Shear/blob/789b229ffaaa71902e329054d9d22803831cd48f/Shear/AppDelegate.swift#L23"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>AppDelegate.swift:23</code></del></td>
    </tr>
  </tbody>
</table>